examples/rack/Gemfile

```diff
@@ -4,5 +4,6 @@ source "https://rubygems.org"
 
 git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
 
-gem "rack"
+gem "rack", ">= 3.0"
+gem "rackup"
```

Comment:
This is what it took to fix the rack example now that it was pulling down Rack v3.

@JamieDanielson, your local environment still looking at `gemfiles/rack.gemfile` seems like the settings Appraisal manipulates for the different named-sets of dependencies are in effect for the old "rack" scenario which—yea—will make everything else in the repo very confused.